.github/workflows/_run-pact-provider-tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Adminusers As Provider Pact Tests
+
+on:
+  workflow_call:
+    inputs:
+      consumer:
+        description: Name of the consumer app, e.g. frontend
```

Comment:
Wonder if it would be better to use "selfservice" or "products-ui", but technically it shouldn't matter